deblur/test/test_workflow.py

```diff
@@ -576,63 +568,65 @@ def test_build_index_sortmerna(self):
-                                    mean_error, error_dist, indel_prob,
-                                    indel_max, trim_length,
-                                    min_size, tuple([ref_fp]), ref_db_fp,
-                                    negate, threads, delim)
-        data = {(0, 0): 9, (1, 0): 9, (2, 0): 10, (3, 0): 8, (4, 0): 2,
-                (5, 0): 2, (6, 0): 8, (7, 0): 2, (8, 0): 8, (9, 0): 9,
-                (10, 0): 7, (11, 0): 9}
-        otu_ids = ["TACCTGCAGCCCAAGTGGTGGTCGATTTTATTGAGTCTAAAACGTTCGTAGCCGG"
-                   "TTTGATAAATCCTTGGGTAAATCGGGAAGCTTAACTTTCCGATTC",
-                   "CACCGGCGGCCCGAGTGGTGGCCATTATTATTGGGTCTAAAGGGTCCGTAGCCGG"
-                   "TTTGATCAGTCTTCCGGGAAATCTGACAGCTCAACTGTTAGGTCT",
-                   "CACCAGCAGCCCAAGTGGTGGCCGGGTTTATTGGGCCTAAAGCGCTCGTAGCCGG"
-                   "ACTGGTAAGTCCCTTGTGAAATCGGTCGGCTCAACCGTTCGGTGC",
-                   "TACCAGCCCCGCGAGTGGTCAGGACGATTATTAAGCCTAAGGCGTCCGTAGCCGG"
-                   "CCTTGTAAGTTCTCCCTTAAATGGTATGGCTCAACCATACCGTGG",
-                   "TTCCGGTCCCAAGCCGAACAGTTTCCGCCGGACGCCCATCCGTTGAGCGGATGGA"
-                   "TTTCCCGACAGACTTGTTCGGCCAGCTACGGACGCTTTAGGCCCA",
-                   "GTGCCAGCCGCCGCGGTAATACCCGCAGCTCAAGTGGTGGTCGCTATTATTGAGC"
-                   "CTAAAACGTCCGTAGTCGGCTTTGTAAATCCCTGGGTAAATCGGG",
-                   "TACCAGCCCCTTAAGTGGTAGGGACGATTATTTGGCCTAAAGCGTCCGTAGCCGG"
-                   "CTTAGTGAGTCTTTCGTTAAATCCGGCGACTTAATCGTCGTTTGC",
-                   "CGCCCGGTCCCAAGCTTTGCAGTCTCCCCAGAAATCGGATCGTTAAGCGACCCGA"
-                   "TTTACCCAGGGATTTACAAAGCCGACTACGGACGTTTTAGGCTCA",
-                   "TACCGGCAGCTCGAGTGATGACCGCTATTATTGGGCCTAAAGCGTCCGTAGCTGG"
-                   "CCGAACAAGTCTGTCGGGAAATCCATCCGCTCAACGGATGGGTCC",
-                   "TACCAGCTCTCCGAGTGGTGGGGACGATTATTGGGCTTAAAGCGTTCGTAGCCGG"
-                   "CTCAACAAGTCTCCCGTTAAATCCAGCGATCTAATCGTTGGATGC",
-                   "TACCAGCTCTCCGAGTGGTGTGGATGTTTATTGGGCCTAAAGCATCCGTAGCTGG"
-                   "CTAGGTTAGTCCCCTGTTAAATCCACCGAATTAATCGTTGGATGC",
-                   "TACCCGCAGCTCAAGTGGTGGTCGCTATTATTGAGCCTAAAACGTCCGTAGTCGG"
-                   "CTTTGTAAATCCCTGGGTAAATCGGGTCGCTTAACGATCCGATTC"]
-        sample_ids = ["s1"]
-        table_exp = Table(data, otu_ids, sample_ids, sample_metadata=None)
-        table_obs = load_table(output_fp)
-        self.assertEqual(table_obs, table_exp)
+        biom_fp = launch_workflow(seqs_fp, output_fp, read_error, mean_error,
+                                  error_dist, indel_prob, indel_max,
+                                  trim_length, min_size, (ref_fp,),
+                                  ref_db_fp, negate, threads, delim)
+
+        # get the trimmed ground truth sequences
+        with open(origfilename, 'U') as f:
+            orig_seqs = [item[1] for item in parse_fasta(f)]
+        orig_seqs = [item[:trim_length].upper() for item in orig_seqs]
```

Comment:
since in general simulated reads can have lowercase, and we don't require
anything about case from the deblurring, so to be on the safe side...

On Thu, Mar 17, 2016 at 4:49 AM, Evguenia Kopylova <notifications@github.com
> wrote:

> In deblur/test/test_workflow.py
> <https://github.com/biocore/deblur/pull/32#discussion_r56491557>:
>
> > -                   "CTAGGTTAGTCCCCTGTTAAATCCACCGAATTAATCGTTGGATGC",
> > -                   "TACCCGCAGCTCAAGTGGTGGTCGCTATTATTGAGCCTAAAACGTCCGTAGTCGG"
> > -                   "CTTTGTAAATCCCTGGGTAAATCGGGTCGCTTAACGATCCGATTC"]
> > -        sample_ids = ["s1"]
> > -        table_exp = Table(data, otu_ids, sample_ids, sample_metadata=None)
> > -        table_obs = load_table(output_fp)
> > -        self.assertEqual(table_obs, table_exp)
> > +        biom_fp = launch_workflow(seqs_fp, output_fp, read_error, mean_error,
> > +                                  error_dist, indel_prob, indel_max,
> > +                                  trim_length, min_size, (ref_fp,),
> > +                                  ref_db_fp, negate, threads, delim)
> > +
> > +        # get the trimmed ground truth sequences
> > +        with open(origfilename, 'U') as f:
> > +            orig_seqs = [item[1] for item in parse_fasta(f)]
> > +        orig_seqs = [item[:trim_length].upper() for item in orig_seqs]
>
> why do you convert the strings to upper case in both orig_seqs and outseqs?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/deblur/pull/32/files/069a0cd16c523f104f10937cd3dd5fd5715c57bb#r56491557>
>